tests/ui/consts/assert-type-intrinsics.stderr

```diff
@@ -1,20 +1,20 @@
-error[E0080]: evaluation of constant value failed
+error[E0080]: evaluation panicked: aborted execution: attempted to instantiate uninhabited type `!`
   --> $DIR/assert-type-intrinsics.rs:11:9
    |
 LL |         MaybeUninit::<!>::uninit().assume_init();
-   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation panicked: aborted execution: attempted to instantiate uninhabited type `!`
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ evaluation of constant value failed
```

Comment:
This note attached to the span also looks very strange to me. It doesn't even point at a constant value.

Reply:
>        // If the current item has generics, we'd like to enrich the message with the
>        // instance and its args: to show the actual compile-time values, in addition to
>        // the expression, leading to the const eval error.

https://github.com/rust-lang/rust/pull/85767

unless there are generics involved, we do not show the path